docs/en/observability/profiling-add-symbols.asciidoc

```diff
@@ -0,0 +1,151 @@
+[[profiling-add-symbols]]
+= Add symbols for native frames
+
+beta::[]
+
+To see function names and line numbers in traces of applications written in programming languages that 
+compile to native code (C, C++, Rust, Go, etc.), you need to push symbols to the cluster.
+
+You can do this with the `elastic-profiling` CLI utility.
+
+Download the `elastic-profiling` binary https://releases.prodfiler.com/elastic-profiling_linux_amd64.tar.gz[here].
+
+NOTE: As the `elastic-profiling` binary is currently only distributed for `Linux x86_64`, you need a Linux machine
+to use it.
+
+[discrete]
+[[profiling-use-elastic-profiling]]
+== How to use the `elastic-profiling` binary
+
+Before using the `elastic-profiling` binary, you need the following information for the deployment where you're running Universal Profiling:
+
+* Credentials (either an API key or username and password) for the `superuser` Elasticsearch role (typically, the `elastic` user).
+* Copy the deployment's Cloud ID from the deployment overview page.

```

Comment:
Since we're referencing the image, and we said earlier "you need the following information", I would rephrase so

```suggestion
* The deployment's Cloud ID from the deployment overview page.
```

I know this is my own text 😅 ... apologies for that